codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/FluentClientDecorator.kt

```diff
@@ -397,10 +397,17 @@ class FluentClientGenerator(
                 val members: List<MemberShape> = input.allMembers.values.toList()
                 val baseDerives = symbolProvider.toSymbol(input).expectRustMetadata().derives
                 val derives = baseDerives.derives.intersect(setOf(RuntimeType.Clone)) + RuntimeType.Debug
-                rust(
+                val operationInput = symbolProvider.toSymbol(input)
+                val okResponse = symbolProvider.toSymbol(operation.outputShape(model))
+                val operationErr = operation.errorSymbol(symbolProvider).toSymbol()
+                rustTemplate(
                     """
                     /// Fluent builder constructing a request to `${operationSymbol.name}`.
                     ///
+                    /// - Takes [`${operationInput.name}`]($operationInput)
```

Comment:
what do you think about enumerating all of the input members here as well with links to the functions?

https://gist.github.com/carllerche/2edccd415957aed257c878fef9e03f25

Reply:
I only avoided it because i wasnt sure how. I'm fine with doing it in this pr or a later one

Reply:
we should iterate over `$operationInput.members()` and their attached documentation